test/built-ins/RegExp/prototype/Symbol.search/set-lastindex-restore.js

```diff
@@ -6,25 +6,29 @@ es6id: 21.2.5.9
 description: The `lastIndex` value is restored following match execution
 info: >
     [...]
-    11. Let status be Set(rx, "lastIndex", previousLastIndex, true).
+    8. If SameValue(currentLastIndex, previousLastIndex) is false, then
+       a. Perform ? Set(rx, "lastIndex", previousLastIndex, true).
     [...]
 features: [Symbol.search]
 ---*/
 
-var latestValue;
+var latestValue = 86;
+var callCount = 0;
 var fakeRe = {
   get lastIndex() {
-    return 86;
+    return latestValue;
```

Comment:
I might be wrong, but by doing this you avoid confirming what the description says: "The `lastIndex` value is restored following match execution"

Now you can't tell `get/set lastIndex` were called